core/jvm/src/test/scala/scalaz/zio/RetrySpec.scala

```diff
@@ -43,9 +43,21 @@ class RetrySpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends Abstrac
     retried must_=== expected
   }
 
-  def retryNJittered = {
-    val retried  = unsafeRun(retryCollect(IO.fail("Error"), Schedule.recurs(5).jittered))
-    val expected = (Left("Error"), List(1, 2, 3, 4, 5).map((Duration.Zero, _)))
+  def retryNUnitIntervalJittered = {
+    val jitter: IO[Nothing, Double] = IO.sync(0.5)
+    val retried = unsafeRun(
+      retryCollect(IO.fail("Error"), Schedule.recurs(5).delayed(_ => 500.millis).jittered(jitter))
```

Comment:
In new tests, probably we should use `schedule.run` to test, instead of the `collect` methods. Not 100% sure on that, though.